src/System.Numerics.Vectors/src/ApiCompatBaseline.uapaot.txt

```diff
@@ -0,0 +1 @@
+MembersMustExist : Member 'System.Numerics.Vector<T>..ctor(System.Span<T>)' does not exist in the implementation but it does exist in the contract.
```

Comment:
Do we still need to add this exception for uapaot?

Reply:
Yes, as of a few hours ago this was needed.  See https://github.com/dotnet/corefx/pull/26499#issuecomment-371609541

Reply:
I added this as I was getting build errors. further discussion [here](https://github.com/dotnet/corefx/pull/26499#issuecomment-371613205).